liveblog/liveblog
# [LBSD-2770] Implement backend for new post's comments feature

### Purpose
This PR introduces the necessary code to handle basic CRUD operations to manage comments for specific posts in Liveblog.

### What has changed
- Added new `PostCommentResource` and `PostCommentService` to store and manage posts' comments
- Added 4 new permissions to allow users to manage posts' comments
- Cleanup tests a little bit
- Added small test case to check if post related items like polls and post comments are fetched using `location` attribute from referenced items

### Steps to test
- Check that everything works as expected in Liveblog admin and frontend 
- Endpoints can be tested using a third party app or service like Postman. Also `curl` can be used to test the new endpoints

Here are a few examples using `curl`. Bear in mind that you need to adjust the Authorisation token, `post_id` and `block_id` value accordingly

Post Comment - Create
```bash
curl --location 'http://127.0.0.1:5001/api/post_comments' \
--header 'Authorization: Basic <REPLACE-TOKEN>' \
--header 'Content-Type: application/json' \
--data '{
    "post_id": "<REPLACE>",
    "blog_id": "<REPLACE>",
    "author_name": "John Doe",
    "text": "Any random comment from myself",
    "is_published": false
}'
```

Post Comment - Read
```bash
curl --location 'http://127.0.0.1:5001/api/post_comments/<REPLACE-COMMENT-ID>' \
--header 'Authorization: Basic <REPLACE-TOKEN>' \
--data ''
```

Post Comment - Delete
```bash
curl --location --request DELETE 'http://127.0.0.1:5001/api/post_comments/<REPLACE-COMMENT-ID>' \
--header 'Authorization: Basic <REPLACE-TOKEN>' \
--header 'If-Match: <REPLACE-E_TAG>' \
--data ''
```

Missing the "update" but you get the idea :) 

Resolves: [LBSD-2770]

Thanks for reviewing!

[LBSD-2770]: https://sofab.atlassian.net/browse/LBSD-2770?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
